src/main/java/uk/ac/ic/wlgitbridge/bridge/repo/GitProjectRepo.java

```diff
@@ -77,7 +73,7 @@ public void useExistingRepository(
     }
 
     @Override
-    public Map<String, RawFile> getFiles()
+    public Map<String, RawFile> getFiles(long maxFileSize)
```

Comment:
Is this function parameter supposed to be used somewhere?

Reply:
I see 4ffe06ac3ec7ca7f7258faf0ad3b7f2383fd1e5b in Part 3 immediately removes this extra parameter, so I'm not sure I see the point in this commit (or PR).